articles/aks/howto-deploy-java-liberty-app.md

```diff
@@ -53,23 +61,38 @@ PASSWORD=$(az acr credential show -n $REGISTRY_NAME --query 'passwords[0].value'
 docker login $LOGIN_SERVER -u $USER_NAME -p $PASSWORD
```

Comment:
@gcharters Using `-o tsv` option which is officially supported by `azure cli` to make the output unquoted.

Reply:
Sounds good.